prow/config/jobs/istio.yaml

```diff
@@ -446,7 +446,7 @@ jobs:
     - --token-path=/etc/github-token/oauth
     - --cmd=go get github.com/envoyproxy/go-control-plane@main && go mod tidy && make clean gen
     requirements: [github]
-    repos: [istio/proxy@master]
+    repos: [istio/test-infra@master]
```

Comment:
Is there a reason to have this as two jobs, and not just have --repo=istio,proxy in the preceding job?

Reply:
fixed